Makefile

```diff
@@ -54,10 +61,10 @@ example_run_direct_table: requirements.txt ## Run the profile directly, job with
 	poetry run python src/ai/whylabs/templates/$(TEMPLATE).py \
 		--job_name="$(JOB_NAME)" \
 		--input-mode=BIGQUERY_TABLE \
-		--input-bigquery-table=bigquery-public-data:hacker_news.comments \
-		--date-column=time_ts \
+		--input-bigquery-sql=bigquery-public-data.hacker_news.full \
+		--date-column=timestamp \
 		--date-grouping-frequency=Y \
-		--org-id=org-0 \
+		--org-id=org-fjx9Rz \
```

Comment:
Are we changing out the org? We'll have to change out the api key too right?

Reply:
I've changed it to test on my personal org (as i don't have access to org-0 in prod)

Reply:
ok, make sure to add me to it as well